app/views/attachments/create.js.erb

```diff
@@ -0,0 +1,6 @@
+<% if @success %>
+  $('#document-modal').modal('hide');
+  location.reload();
+<% else %>
+  location.reload();
+<% end %>
```

Comment:
if we are reloading location in both cases they why we are using if else condition.